substrate/frame/election-provider-multi-block/src/lib.rs

```diff
@@ -1682,12 +1683,12 @@ mod phase_rotation {
 					}],
 				);
 
-				roll_to(24);
+				roll_to(25);
```

Comment:
Good reason to stick to `roll_to_this_signed` in the future. These tests are reasonable, as we _intentionally want_ to manually check them.